### ISSUE TITLE
Boots the vivint plugin then crashes shortly 

### ISSUE THREAD

**Issue by Jeffrey345678** [balansse/homebridge-vivint#8]
[10/3/2020, 2:38:19 PM] [Config] Terminal session ended.
[10/3/2020, 2:38:19 PM] [HB Supervisor] Restarting Homebridge...
[10/3/2020, 2:38:19 PM] [HB Supervisor] Starting Homebridge with extra flags: -I -P /homebridge/node_modules
[10/3/2020, 2:38:19 PM] [HB Supervisor] Started Homebridge v1.2.3 with PID: 731
[10/3/2020, 2:38:20 PM] Loaded config.json with 0 accessories and 3 platforms.
[10/3/2020, 2:38:20 PM] ---
[10/3/2020, 2:38:20 PM] Loaded plugin: homebridge-camera-ffmpeg@3.0.3
[10/3/2020, 2:38:20 PM] Registering platform 'homebridge-camera-ffmpeg.Camera-ffmpeg'
[10/3/2020, 2:38:20 PM] ---
[10/3/2020, 2:38:20 PM] Loaded plugin: homebridge-dummy@0.4.1
[10/3/2020, 2:38:20 PM] Registering accessory 'homebridge-dummy.DummySwitch'
[10/3/2020, 2:38:20 PM] ---
[10/3/2020, 2:38:20 PM] Loaded plugin: @balansse/homebridge-vivint@1.0.1
[10/3/2020, 2:38:20 PM] Registering platform '@balansse/homebridge-vivint.Vivint'
[10/3/2020, 2:38:20 PM] ---
[10/3/2020, 2:38:20 PM] Loaded plugin: homebridge-config-ui-x@4.28.0
[10/3/2020, 2:38:20 PM] Registering platform 'homebridge-config-ui-x.config'
[10/3/2020, 2:38:20 PM] ---
[10/3/2020, 2:38:20 PM] Loading 3 platforms...
[10/3/2020, 2:38:20 PM] [Config] Initializing config platform...
[10/3/2020, 2:38:20 PM] [Config] Running in Service Mode
[10/3/2020, 2:38:20 PM] [Vivint] Initializing Vivint platform...
[10/3/2020, 2:38:20 PM] [Camera FFmpeg] Initializing Camera-ffmpeg platform...
[10/3/2020, 2:38:20 PM] [Camera FFmpeg] [Doorbell ] Configuring cached bridged accessory...
[10/3/2020, 2:38:20 PM] [Camera FFmpeg] [West Camera] Configuring cached bridged accessory...
Setup Payload:
X-HM://0023ISYWYJ8O8
Enter this code with your HomeKit app on your iOS device to pair with Homebridge:
                       
    ┌────────────┐     
    │ 031-45-154 │     
    └────────────┘     
                       
[10/3/2020, 2:38:20 PM] Homebridge v1.2.3 is running on port 52082.
[10/3/2020, 2:38:23 PM] [Vivint] Device not (yet) supported [ID]:5 [Type]:energy_service [EquipmentCode]:undefined [Name]:undefined
[10/3/2020, 2:38:23 PM] [Vivint] Device not (yet) supported [ID]:11 [Type]:network_hosts_service [EquipmentCode]:undefined [Name]:undefined
[10/3/2020, 2:38:23 PM] [Vivint] Device not (yet) supported [ID]:12 [Type]:panel_diagnostics_service [EquipmentCode]:undefined [Name]:undefined
[10/3/2020, 2:38:23 PM] [Vivint] Ignored device [ID]:13 [Type]:primary_touch_link_device [EquipmentCode]:undefined [Name]:Panel
[10/3/2020, 2:38:23 PM] [Vivint] Ignored device [ID]:23 [Type]:wireless_sensor [EquipmentCode]:1251 [Name]:Middle East Basement Window
[10/3/2020, 2:38:23 PM] [Vivint] Ignored device [ID]:22 [Type]:wireless_sensor [EquipmentCode]:1251 [Name]:Left East Basement Window
[10/3/2020, 2:38:23 PM] [Vivint] Ignored device [ID]:24 [Type]:wireless_sensor [EquipmentCode]:1251 [Name]:Right East Basement Window
[10/3/2020, 2:38:23 PM] [Vivint] Ignored device [ID]:25 [Type]:wireless_sensor [EquipmentCode]:1251 [Name]:West Basement Window
[10/3/2020, 2:38:23 PM] [Vivint] Ignored device [ID]:31 [Type]:wireless_sensor [EquipmentCode]:1251 [Name]:Sliding Glass Door
[10/3/2020, 2:38:23 PM] [Vivint] Ignored device [ID]:33 [Type]:wireless_sensor [EquipmentCode]:1252 [Name]:Front Door
[10/3/2020, 2:38:23 PM] [Vivint] Ignored device [ID]:35 [Type]:wireless_sensor [EquipmentCode]:1251 [Name]:Left Entry Window
[10/3/2020, 2:38:23 PM] [Vivint] Ignored device [ID]:34 [Type]:wireless_sensor [EquipmentCode]:1252 [Name]:Garage Door
[10/3/2020, 2:38:23 PM] [Vivint] Ignored device [ID]:37 [Type]:wireless_sensor [EquipmentCode]:1251 [Name]:Front Bedroom Window
[10/3/2020, 2:38:23 PM] [Vivint] Ignored device [ID]:40 [Type]:wireless_sensor [EquipmentCode]:1251 [Name]:Left Living Room Window
[10/3/2020, 2:38:23 PM] [Vivint] Ignored device [ID]:41 [Type]:wireless_sensor [EquipmentCode]:1251 [Name]:Right Living Room Window
[10/3/2020, 2:38:23 PM] [Vivint] Device not (yet) supported [ID]:51 [Type]:slim_line_device [EquipmentCode]:undefined [Name]:Glance
[10/3/2020, 2:38:23 PM] [Vivint] Device not (yet) supported [ID]:60 [Type]:wireless_sensor [EquipmentCode]:475 [Name]:North East Basement Glass Break
[10/3/2020, 2:38:23 PM] [Vivint] Device not (yet) supported [ID]:62 [Type]:wireless_sensor [EquipmentCode]:475 [Name]:South West Basement Glass Break
[10/3/2020, 2:38:23 PM] [Vivint] Ignored device [ID]:65 [Type]:wireless_sensor [EquipmentCode]:609 [Name]:Kitchen Motion Detector
[10/3/2020, 2:38:23 PM] [Vivint] Ignored device [ID]:66 [Type]:wireless_sensor [EquipmentCode]:609 [Name]:Front Entry Motion sensor
[10/3/2020, 2:38:23 PM] [Vivint] Device not (yet) supported [ID]:67 [Type]:wireless_sensor [EquipmentCode]:475 [Name]:Front Entry Glass Break
[10/3/2020, 2:38:23 PM] [Vivint] Device not (yet) supported [ID]:69 [Type]:wireless_sensor [EquipmentCode]:475 [Name]:Master Bedroom Glass Break
[10/3/2020, 2:38:23 PM] [Vivint] Camera [Doorbell] configuration: {
    "name": "Doorbell",
    "manufacturer": "VIVOTEK",
    "model": "DB8332SW",
    "videoConfig": {
        "source": "-rtsp_transport tcp -re -i rtsp://user:s_icdSD9MWaBXFJhOPcmEKI@10.0.0.72:8554/Video-71",
        "vcodec": "copy",
        "audio": true
    }
}
[10/3/2020, 2:38:23 PM] [Vivint] Device not (yet) supported [ID]:72 [Type]:iot_service [EquipmentCode]:undefined [Name]:undefined
[10/3/2020, 2:38:23 PM] [Vivint] Device not (yet) supported [ID]:74 [Type]:scheduler_service [EquipmentCode]:undefined [Name]:undefined
[10/3/2020, 2:38:23 PM] [Vivint] Device not (yet) supported [ID]:176 [Type]:yofi_device [EquipmentCode]:undefined [Name]:AirBridge-176
[10/3/2020, 2:38:23 PM] [Vivint] Device not (yet) supported [ID]:180 [Type]:lgit_poe_wifi_device [EquipmentCode]:undefined [Name]:East Camera-Bridge
[10/3/2020, 2:38:23 PM] [Vivint] Camera [West Camera] configuration: {
    "name": "West Camera",
    "manufacturer": "VIVOTEK",
    "model": "HDP450",
    "videoConfig": {
        "source": "-rtsp_transport tcp -re -i rtsp://user:s_icdSD9MWaBXFJhOPcmEKI@10.0.0.72:8554/Video-182",
        "vcodec": "copy",
        "audio": true
    }
}
[10/3/2020, 2:38:23 PM] [Vivint] Device not (yet) supported [ID]:183 [Type]:lgit_poe_wifi_device [EquipmentCode]:undefined [Name]:West Camera-Bridge
[10/3/2020, 2:38:23 PM] [Vivint] Removing 0 stale accessories
[10/3/2020, 2:38:23 PM] [Vivint] Adding 0 new accessories
[10/3/2020, 2:38:23 PM] [Vivint] Error while bootstrapping accessories: TypeError: Cannot set property 'Status' of undefined
    at DeviceSet.handleSnapshot (/homebridge/node_modules/@balansse/homebridge-vivint/lib/device_set.js:59:24)
    at /homebridge/node_modules/@balansse/homebridge-vivint/index.js:81:25
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[10/3/2020, 2:38:23 PM] Error: This callback function has already been called by someone else; it can only be called one time.
    at /usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/util/once.ts:6:13
    at Characteristic.<anonymous> (/homebridge/node_modules/@balansse/homebridge-vivint/index.js:115:15)
    at Characteristic.emit (events.js:327:22)
    at Characteristic.EventEmitter.emit (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/EventEmitter.ts:42:22)
    at Characteristic._this.getValue (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Characteristic.ts:489:12)
    at /usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Accessory.ts:1239:22
    at Array.forEach (<anonymous>)
    at Bridge.Accessory._this._handleGetCharacteristics (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Accessory.ts:1167:10)
    at HAPServer.emit (events.js:315:20)
    at HAPServer.EventEmitter.emit (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/EventEmitter.ts:42:22)
[10/3/2020, 2:38:23 PM] Got SIGTERM, shutting down Homebridge...
[10/3/2020, 2:38:28 PM] [HB Supervisor] Homebridge Process Ended. Code: 143, Signal: null

**Comment by balansse** [comment]
I see you have configured to ignore "primary_touch_link_device" and multiple "wireless_sensor" devices - is there a specific reason for this, or is it a bug? 

I also see that Glass Breaks are reported "not supported" by the plugin but they actually are. I suspect outdated cache might cause this problem - could you try cleaning .homebridge/cachedAccessories folder and try again?

**Comment by Jeffrey345678** [comment]
I don't want the regular sensors showing up because I have 35 or more sensors and that's just more functionality that I don't want to build automation for right now. Unneeded overhead for me at least. Also, after only about a year old most of my sensors either have bad batteries or tamper issues. One day i'll hardwire everything with a few take345 or something. I enabled and disabled wireless sensor devices and left primary touch link on. Seems to be working great now. The whole issue was probably because I restored from a past config file!
thanks, Jeffrey
